samples/dashboard/demos/climate/demo.css

```diff
@@ -58,51 +174,89 @@
  *
  * */
 
+#world-map > div > div {
+    background: var(--color-background);
+}
+
 #world-map .highcharts-background {
-    fill: #46475d;
+    fill: var(--color-background);
+}
+
+#world-map .highcharts-credits {
+    fill: var(--color-map);
+}
+
+#world-map .highcharts-halo {
+    display: none;
+}
+
+#world-map .highcharts-map-navigation {
+    fill: var(--color-navigation);
+    font-size: 20px;
```

Comment:
Can we use font-size in em / rem instead of static px? It is important for zooming in/out page.

Reply:
This font size is optimized for the rect shape and needs to be pixel.